virtcontainers/qemu.go

```diff
@@ -638,6 +638,11 @@ func (q *qemu) setupVirtiofsd() (err error) {
 	var listener *net.UnixListener
 	var fd *os.File
 
+	_, err = os.Stat(q.config.VirtioFSDaemon)
+	if err != nil {
```

Comment:
`os.Stat` can fail for different reasons, I think here you want to check if the file exist, isn't it?

```go
if os.IsNotExist(err) {
```

https://golang.org/pkg/os/#IsNotExist

Reply:
Fixed!